dir.props

```diff
@@ -117,6 +117,28 @@
   </PropertyGroup>
 
   <Import Project="$(BuildConfigurationImportFile)" Condition="Exists('$(BuildConfigurationImportFile)')" />
+  <PropertyGroup>
+    <HostOS>$(RuntimeOS)</HostOS>
+    <HostOS Condition="'$(HostOS)' == 'tizen.4.0.0'">ubuntu.14.04</HostOS>
+    <HostRid>$(HostOS)-x64</HostRid>
+
+    <_runtimeOSVersionIndex>$(RuntimeOS.IndexOfAny(".-0123456789"))</_runtimeOSVersionIndex>
+    <_runtimeOSFamily Condition="'$(_runtimeOSVersionIndex)' != '-1'">$(RuntimeOS.SubString(0, $(_runtimeOSVersionIndex)))</_runtimeOSFamily>
+    <_portableRID>linux</_portableRID>
+    <_portableRID Condition="'$(_runtimeOSFamily)' == 'win'">win</_portableRID>
+    <_portableRID Condition="'$(_runtimeOSFamily)' == 'osx'">osx</_portableRID>
+<!--
+    Why doesn't this work?
```

Comment:
This is the "AllConfigurations" build concept.  We need to maintain that build, but it doesn't need to support TargetRID.  TargetRID concept clashes with "AllConfigurations" concept.